docs/get-started/install.rst

```diff
@@ -25,7 +25,7 @@ virtualenv
 
 Install virtualenv::
 
-    pip install virtualenv
+    pip3 install virtualenv
```

Comment:
Why not just use the Python Standard Library's https://docs.python.org/3/library/venv.html instead of installing virtualenv?

Reply:
`venv` is probably a good idea. I use `virtualenv` out of habit. Does it offer any advantage over `venv`?

Reply:
The first paragraph of https://virtualenv.pypa.io explains the differences.